### DISCUSSION TITLE
Focus order inside the tarteaucitron modal since the save button

### DISCUSSION THREAD

**Discussion by louismaximepiton** [AmauriC/tarteaucitron.js#1280] [Q&A]
Hello there,

Thanks a lot for this time saving project to be here and maintained 🙏 

The save button have been added [here](https://github.com/AmauriC/tarteaucitron.js/commit/c963fa88271d4890e13b6e5473475ac6fc16e6f6).

<details>
<summary>Here is the final rendering and focus order I have now</summary>

![image](https://github.com/user-attachments/assets/ea3380bf-2715-4fce-b614-3e381612c807)

</details>

I may have several questions concerning the focus order inside the `panel`:
- Why is the SelfLink inside the `tarteaucitronBorder`? IMO, from what I understand, it would make sense to have it after the save link in the HTML.
- Is there any way to change the focus trap using the `tarteaucitron.userInterface` and so avoid coding it from scratch?
- I can't yet remove safely `#tarteaucitronSave` element since it is referenced as `lastFocusableEl` inside the `focusTrap()` and so I lose focus trap.

Do you see any workaround ?
I'd be happy to contribute on this if needed.

**Comment by AmauriC** [comment]
Hi,

For the self link you are right, I note to change the position.

If you set a `visibility:hidden` to the save button it doesn't work?
